pkg/crc/machine/start.go

```diff
@@ -331,6 +332,20 @@ func (client *client) Start(ctx context.Context, startConfig types.StartConfig)
 			return nil, fmt.Errorf("Failed to rotate bearer token for cockpit webconsole: %w", err)
 		}
 
+		p := podman.UsePodman()
+		c, err := client.ConnectionDetails()
+		if err != nil {
+			return nil, err
+		}
+		rootlessURI := fmt.Sprintf("ssh://%s@%s:%d%s", c.SSHUsername, c.IP, c.SSHPort, constants.RootlessPodmanSocket)
+		rootfulURI := fmt.Sprintf("ssh://%s@%s:%d%s", c.SSHUsername, c.IP, c.SSHPort, constants.RootfulPodmanSocket)
+		if err := p.AddRootlessSystemConnection(c.SSHKeys[0], rootlessURI); err != nil {
+			return nil, err
+		}
+		if err := p.AddRootfulSystemConnection(c.SSHKeys[0], rootfulURI); err != nil {
+			return nil, err
+		}
+
```

Comment:
This deserves its own helper (`addPodmanSystemConnections()` or something like this) to make it obvious to the reader what this is about without overwhelming it with implementation details.

Reply:
Done.